_content/articles/century-of-confusion_green-phillip-scott-ellis.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "A Century of Confusion: The Brick Reliefs of Cambodia's Phnom Trap Towers"
+authors:
+  - "Green, Phillip Scott Ellis"
+external_url: "https://buddhism.lib.ntu.edu.tw/en/search/search_detail.jsp?seq=607765&comefrom=authorinfo"
+drive_links:
+  - "https://drive.google.com/file/d/1sA_-0d0MZRSDewN1h81AanSi8kV3-61o/view?usp=sharing"
+course: cambodian
+tags:
+  - bart
+  - architecture
```

Comment:
Instead of these two, let's do "buddhist-architecture" specifically and "medieval"